src/js/form-util.js

```diff
@@ -78,8 +123,40 @@ export function getReasons (reasonInputs) {
   return reasons
 }
 
-export function prepareInputs (formInputs, reasonInputs, reasonFieldset, reasonAlert, snackbar) {
+export function getReasonsObject (reasonInputs) {
+  const reasons = reasonInputs
+    .filter((reason) => reason.checked)
+    .reduce(function (map, reason) {
+      map[reason.value] = reason.checked
+      return map
+    }, {})
+  return reasons
```

Comment:
Proposition : la fonction peut être inlinée (variable `reasons` inutile), et pour plus d'homogénéité, on peut tout passer en arrow function.

```suggestion
  return reasonInputs
    .filter((reason) => reason.checked)
    .reduce((map, reason) => {
      map[reason.value] = reason.checked
      return map
    }, {})
```